share/rocm/cmake/ROCMSetupVersion.cmake

```diff
@@ -154,7 +154,18 @@ endfunction()
 
 function(rocm_set_soversion LIBRARY_TARGET SOVERSION)
     if(NOT WIN32)
-        rocm_version_regex_parse("^([0-9]+).*" SOVERSION_MAJOR "${SOVERSION}")
-        set_target_properties(${LIBRARY_TARGET} PROPERTIES VERSION ${SOVERSION} SOVERSION ${SOVERSION_MAJOR})
+        rocm_version_regex_parse("^([0-9]+).*" LIB_VERSION_MAJOR "${SOVERSION}")
+        rocm_version_regex_parse("^[0-9]+\\.([0-9]+).*" LIB_VERSION_MINOR "${SOVERSION}")
+
+        if (DEFINED ENV{ROCM_LIBPATCH_VERSION})
```

Comment:
I think a name like `ROCM_SOPATCH_VERSION` would make it clearer this is setting the SO version.

Reply:
This name is being set in the jenkins build environment and being agreed upon by other teams.
One thing I can do is to set ROCM_SOPATCH_VERSION as cache variable and pass the value as a parameter while building the package. Let me know if this fine.

Reply:
Please review the latest commit.